src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -179,12 +179,11 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 	private _labelComputer?: TerminalLabelComputer;
 	private _userHome?: string;
 	private _hasScrollBar?: boolean;
+	private _target?: TerminalLocation | undefined;
 
-	get target(): TerminalLocation | undefined { return this.xterm?.target; }
+	get target(): TerminalLocation | undefined { return this._target; }
 	set target(value: TerminalLocation | undefined) {
-		if (this.xterm) {
-			this.xterm.target = value;
-		}
+		this._target = value;
```

Comment:
We still need to be able to set the target after creation, let's also do `this.xterm.target = value` here and keep them in sync